hca2scea-backend/test/golden/TEST02/E-HCAD-36.idf.txt

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I would like to suggest a differnt naming convention for the test data.
Please name the spreadsheet and the directory containing the idf and sdrf using the accession number.
In this case:
- spreadsheet: SRP274475.xlsx
- idf & sdrf should be in `test/golden/SRP274475`

Reply:
Ok will do.